### PR TITLE
add 35. Search Insert Position

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 35.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Solution(object):
+    def searchInsert(self, nums: List[int], target: int) -> int:
+        """A function to  return the index if the target is found. If not, return the index where it would be if it were inserted in order.
+        Args:
+            nums (List[int]): A given a sorted array of distinct integers.
+            target (int): A given an integer.
+        Returns:
+            int: The index if the target is found. If not, return the index where it would be if it were inserted in order.
+        """
+        r=0
```
Comment (kobakobashu) [inline 35.py:14]: r = 0
https://www.flake8rules.com/rules/E225.html

File: 35.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        l=len(nums)-1
```
Comment (kobakobashu) [inline 35.py:15]: The variable name "l" is inappropriate
https://www.flake8rules.com/rules/E741.html#:~:text=Variables%20named%20I%20%2C%20O%20%2C%20and,variables%20to%20something%20more%20descriptive.

File: 35.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# n = len(nums)
+# time:O(n)
```
Comment (kobakobashu) [inline 35.py:2]: In a binary search, the search area is halved, so it can be computed in log(n).

**Review by kobakobashu — CHANGES_REQUESTED** [review]
Please use PEP8, and you will understand what I mean.
https://note.com/npaka/n/n6afc09f45126

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request merged**